mempool/src/tests/common.rs

```diff
@@ -142,9 +142,7 @@ pub(crate) fn batch_add_signed_txn(
     transactions: Vec<SignedTransaction>,
 ) -> Result<()> {
     for txn in transactions.into_iter() {
-        if let Err(e) = add_signed_txn(pool, txn) {
-            return Err(e);
-        }
+        add_signed_txn(pool, txn)?;
```

Comment:
:/ this kinda reads much worse imo

Very easy to miss the ?

Reply:
This is much more idiomatic imo, most everyone who codes Rust knows to look out for `?`. Perhaps a comment to clear it up.

Reply:
@banool  @gregnazario   This is the only way to pass the test without changing the rules of lint